source/common/upstream/upstream_impl.cc

```diff
@@ -641,6 +688,24 @@ ClusterInfoImpl::ClusterInfoImpl(const envoy::api::v2::Cluster& config,
   // https://github.com/lyft/protoc-gen-validate/issues/97 resolved. This just provides early
   // validation of sanity of fields that we should catch at config ingestion.
   DurationUtil::durationToMilliseconds(common_lb_config_.update_merge_window());
+
+  // Create upstream filter factories
+  auto filters = config.filters();
+  for (ssize_t i = 0; i < filters.size(); i++) {
+    const auto& proto_config = filters[i];
+    const std::string& string_name = proto_config.name();
+    ENVOY_LOG(debug, "  upstream filter #{}:", i);
```

Comment:
What does this debug output look like? It's printed for every cluster so just want to make sure it's useful.

Reply:
Here is a sample:
```log
[2019-07-25 22:02:21.833][26][info][config] [source/server/configuration_impl.cc:68] loading 1 cluster(s)
[2019-07-25 22:02:21.833][27][debug][grpc] [source/common/grpc/google_async_client_impl.cc:43] completionThread running
[2019-07-25 22:02:21.839][26][debug][upstream] [source/common/upstream/upstream_impl.cc:695]   upstream filter #0:
[2019-07-25 22:02:21.839][26][debug][upstream] [source/common/upstream/upstream_impl.cc:696]     name: envoy.upstream.polite
[2019-07-25 22:02:21.839][26][debug][upstream] [source/common/upstream/cluster_manager_impl.cc:838] adding TLS initial cluster cluster_0
[2019-07-25 22:02:21.840][26][debug][upstream] [source/common/upstream/upstream_impl.cc:845] initializing secondary cluster cluster_0 completed
```